epsagon/utils.py

```diff
@@ -137,3 +139,22 @@ def import_original_module():
         return __import__(module_path), module_path, handler_name
     except ImportError:
         raise ImportError('Failed to import module: {}'.format(module_path))
+
+
+def collect_container_metadata():
+    """
+    Collects container metadata if exists.
+    :return: dict.
+    """
+    metadata_uri = os.environ.get('ECS_CONTAINER_METADATA_URI')
+    if not metadata_uri:
+        return {}
+    container_metadata = json.loads(requests.get(metadata_uri).content)
+
+    # Remove event from events list
+    events = list(trace_factory.get_trace().events_map.keys())
```

Comment:
I think we should remove only event with the correct type and uri (this might be an issue in the future if some other operation will be inserted before)